tests/test_nn/test_init.py

```diff
@@ -116,3 +116,16 @@ def setUp(self) -> None:
         )
         self.num_entities = dataset.num_entities
         self.shape = self.initializer.tensor.shape[1:]
+
+
+class RandomWalkPositionalEncodingInitializerTestCase(cases.InitializerTestCase):
+    """Tests for random-walk positional encoding."""
+
+    def setUp(self) -> None:
```

Comment:
any chance of giving a few concrete examples in the tests?

Reply:
concrete = manually designed graphs with expected numeric values?

Reply:
yes, sorry, I know that's a lot of work tho

Reply:
I took an example from the paper and put in the test - right now it fails at the `assert` line 🤔 

<img width="424" alt="Screen Shot 2022-05-10 at 12 55 46 PM" src="https://user-images.githubusercontent.com/4430986/167682270-1afc7add-28c7-42e7-af84-bcf6987758c4.png">

Reply:
ah I know why - because we are doing sparse matrix multiplication and those values are expected to be zeros